### DISCUSSION TITLE
[Beginner Question] Text Cursor/Caret Symbol — Change from Block (`█`) to Beam (`▏`)

### DISCUSSION THREAD

**Discussion by Humboldt-Penguin** [charmbracelet/bubbletea#1090] [Q&A] [answered]
Hi, hope everyone is doing alright.

**My Question** — For text-based inputs, is it possible to make the cursor/caret appear as a vertical beam (`|` or `▏`) as opposed to the default block character (`█`)? I think this depends on whether the cursor is its own rendered character, or if it simply inverts colors on the current cell.

For context, I'm working on a FOSS/self-hosted alternative to paid/proprietary apps like Workflowy/Dynalist in the "infinite outliner" niche, and `bubbletea` is a huge help!

Thank you for your time, hope you have a nice day :)

**Comment by meowgorithm** [comment, marked as answer]
Hi! This isn’t possible yet but support for this is currently in active development. Right now the cursor is simulated and we’re making renderer updates to support a true cursor.

**Reply by Humboldt-Penguin** [reply]
Thank you so much for the prompt and helpful answer! Take care
